docs/src/modules/java-protobuf/pages/running-locally.adoc

```diff
@@ -0,0 +1,266 @@
+= Run a service locally
+:page-supergroup-java-scala: Language
+include::ROOT:partial$include.adoc[]
+
+Running a service locally is helpful to test and debug. The following sections provide commands for starting and stopping a single service locally.
+
+== Prerequisites
+
+In order to run your service locally, you'll need to have the following prerequisites:
+
+* Docker {minimum_docker_version} or higher
+* Access to the `gcr.io/kalix-public` container registry
+
+[TIP]
+====
+The samples provided with the Kalix SDKs all have `docker-compose` files preconfigured to run your service locally. They also contain the configuration to start a local Google Pub/Sub emulator that the Kalix proxy will connect to when you make use of topic-based Eventing in your service.
```

Comment:
Didn't we comment out the pub/sub from samples not using topics, or was that only something discussed?

Reply:
yes, I will remove this.